app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const fs = require("fs/promises");
+const { getAllTopics } = require('./controllers/topics.controllers');
+
+const app = express();
+app.use(express.json());
```

Comment:
Are we using this line of code at the moment? if not, please remove until necessary 🙂